code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -259,3 +264,56 @@
 	New()
 		..()
 		reagents.add_reagent("synthanol", 50)
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/bottler
+	name = "generic beverage container"
+	desc = "this shouldn't ever be spawned. shame on you"
+	icon_state = "glass_bottle"
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/bottler/on_reagent_change()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/bottler/pickup(mob/user)
+	..()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/bottler/dropped(mob/user)
+	..()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/bottler/attack_hand()
+	..()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/food/drinks/cans/bottler/update_icon()
+	overlays.Cut()
+
+	if(reagents.total_volume)
+		var/image/filling = image('icons/obj/reagentfillings.dmi', src, "[icon_state]10")
+
+		switch(reagents.total_volume)
+			if(0 to 9)		filling.icon_state = "[icon_state]-10"
```

Comment:
all of these need to be properly indented on the next line and not done like this.